drivers/virtualbox/virtualbox.go

```diff
@@ -131,27 +131,37 @@ func (d *Driver) Create() error {
 	if err != nil {
 		return err
 	}
+	ignoreISOUpdate := false
 
-	if d.Boot2DockerURL != "" {
-		isoURL = d.Boot2DockerURL
-		log.Infof("Downloading boot2docker.iso from %s...", isoURL)
-		if err := utils.DownloadISO(d.storePath, "boot2docker.iso", isoURL); err != nil {
-			return err
-		}
-	} else {
-		// todo: check latest release URL, download if it's new
-		// until then always use "latest"
-		isoURL, err = utils.GetLatestBoot2DockerReleaseURL()
-		if err != nil {
-			return err
+	// I have to ingnore ISO download if cmd has --ignoreISOUpdate
+	for _, f := range os.Args {
```

Comment:
This should use the args in `commands.go` to add another arg for the virtualbox driver such as `--virtualbox-skip-update` or similar.